Resources/Prototypes/Roles/Jobs/Security/brigmedic.yml

```
@@ -0,0 +1,53 @@
+    - !type:RoleTimeRequirement
+      role: JobSecurityOfficer
+      time: 18000 #5 hrs
+    - !type:RoleTimeRequirement
+      role: JobChiefMedicalOfficer
+      time: 18000 #5 hrs
+  startingGear: BrigmedicGear
+  icon: "JobIconBrigmedic"
+  supervisors: job-supervisors-hos
+  canBeAntag: false
+  access:
+  - Security
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)